src/main/java/core/basesyntax/DrawInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawInfo {
+    void toDraw();
```

Comment:
The method name `toDraw` in the interface is a verb phrase, which is not recommended. Method names should be actions, so a better name for this method would be simply `draw` to directly represent the action it performs.